client/src/components/JobList.vue

```diff
@@ -101,19 +125,14 @@
   .job-results__list {
     padding: 0;
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
Not sure about that, just wanted to center the list when only 3 cards or less are displayed in a row (= when screen width gets smaller). Otherwise all cards were aligned to the left.

Reply:
I need to fork your repo, to give it a try. Don't have enough time right now, but I will do it soon.